cloudca/resource_cloudca.go

```diff
@@ -65,3 +66,17 @@ func getResources(d *schema.ResourceData, meta interface{}) cloudca.Resources {
 	_resources, _ := client.GetResources(d.Get("service_code").(string), d.Get("environment_name").(string))
 	return _resources.(cloudca.Resources)
 }
+
+// Deals with all of the casting done to get a cloudca.Resources.
+func getResourcesForEnvironmentId(d *schema.ResourceData, meta interface{}) (cloudca.Resources, error) {
+	client := meta.(*cca.CcaClient)
+	environment, err := client.Environments.Get(d.Get("environment_id").(string))
+	if err != nil {
+		return cloudca.Resources{}, err
+	}
+	resources, err2 := client.GetResources(environment.ServiceConnection.ServiceCode, environment.Name)
```

Comment:
If I remember correctly, `err2` can just be `err`. No need to introduce a new variable.